common/authz_tx_types.go

```diff
@@ -7,6 +7,7 @@ const (
 	OutboundVoter TxType = "OutboundVoter"
 	NonceVoter    TxType = "NonceVoter"
 	GasPriceVoter TxType = "GasPriceVoter"
+	RefundAborted TxType = "RefundAborted"
```

Comment:
f7b9e05286588675cf2bd66469003e52def763c5

Move to the message file, but as as mentioned in the other comment, it might be a good idea to move all the txTypes strings into the common package